frontend/app/routes/public/renew/$id/ita/demographic-survey.tsx

```diff
@@ -219,7 +219,7 @@ export default function RenewItaDemographicSurveyQuestions() {
             <p>{t('renew-ita:demographic-survey.improve-cdcp')}</p>
             <p>{t('renew-ita:demographic-survey.confidential')}</p>
             <p>{t('renew-ita:demographic-survey.impact-enrollment')}</p>
-            <Button name="_action" value={FormAction.Save} variant="alternative" endIcon={faChevronRight} data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application:Prefer not to answer - Demographic survey click">
+            <Button name="_action" value={FormAction.Save} variant="alternative" endIcon={faChevronRight} data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-ITA:Prefer not to answer - Demographic survey click">
```

Comment:
To match the page title and to be consistent with other flows:
```suggestion
            <Button name="_action" value={FormAction.Save} variant="alternative" endIcon={faChevronRight} data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-ITA:Prefer not to answer - Voluntary demographic questions click">
```

(Same comment for the rest of the attribute values in this route)